### ISSUE TITLE
Another reason why images instead of text is bad

### ISSUE THREAD

**Issue by carlosmunozrodriguez** [WillSullivan/idownvotedbecause#30]
[Image of and exception](http://idownvotedbecau.se/imageofanexception/) and [Images of code](http://idownvotedbecau.se/imageofcode) should include as a reason why this is bad the fact that if a person with visual impairment is using a screen reader to read the question, he will be unable to get the screen reader to read the image.

**Comment by carlosmunozrodriguez** [comment]
Sorry this is already steated
